pkg/reconciler/instances/istio/action.go

```diff
@@ -24,6 +24,36 @@ type ReconcileAction struct {
 	performer actions.IstioPerformer
 }
 
+func NewReconcileAction(performer actions.IstioPerformer) *ReconcileAction {
+	return &ReconcileAction{performer: performer}
+}
+
+type UninstallAction struct {
+	performer actions.IstioPerformer
+}
+
+func NewUninstallAction(performer actions.IstioPerformer) *UninstallAction {
```

Comment:
Do we really need this constructor? If so, please add documentation. It is an exposed function.

Reply:
I would say so yes, and I added a small docu